src/pre-engagement-form/index.tsx

```diff
@@ -57,7 +78,9 @@ const PreEngagementForm: React.FC<Props> = ({ formState: defaultValues, formDefi
       <LocalizationProvider manager={manager}>
         <form className="Twilio-DynamicForm" onSubmit={onSubmit}>
           <Title title={formDefinition.description} />
-          {generateForm(formDefinition.fields)}
+          <ReCAPTCHA sitekey={RECAPTCHA_SITE_KEY} size="invisible" ref={recaptchaRef}>
```

Comment:
To change between the different options with v2, size in ReCAPTCHA can be changed to invisible, checkbox, or normal.